apps/mobile-wallet/index.ts

```diff
@@ -2,9 +2,7 @@ import './shim'
 import '~/features/localization/i18n'
 import './setupErrorHandling'
 // Node polyfills to be removed when dependencies are updated
-import 'react-native-get-random-values'
 import 'react-native-url-polyfill/auto'
```

Comment:
The import is still here although the dep has been removed?
(maybe fixed in a followup commit, will check).